pkg/operator/ceph/operator.go

```diff
@@ -56,10 +56,11 @@ var provisionerConfigs = map[string]string{
 }
 
 var (
-	// Whether to enable the flex driver. If true, the rook-ceph-agent daemonset will be started.
-	EnableFlexDriver = true
-	// Whether to enable the daemon for device discovery. If true, the rook-ceph-discover daemonset will be started.
-	EnableDiscoveryDaemon = true
+	// EnableFlexDriver Whether to enable the flex driver. If true, the rook-ceph-agent daemonset will be started.
+	EnableFlexDriver bool
```

Comment:
This field is set to `false` by translating the operator env var to the command line [here](https://github.com/rook/rook/blob/master/cmd/rook/ceph/operator.go#L46). I wasn't able to repro the issue of the flex agent always starting. Shall we close this issue? Or how did you repro the isuse?